Add docs for new istioctl and operator change (#7791)
istio/istio.io: content/en/docs/setup/install/istioctl/index.md

## Uninstall Istio

To uninstall Istio completely, run the following command:

{{< text bash >}}

> $ istioctl x uninstall <your original installation options> --purge

Reviewer: Is it required to pass `<your original installation options>`?

I think just passing `istioctl x uninstall --purge` will remove all control planes and shared resources?

Author: yep it is not required actually, removed it

{{< /text >}}

> The optional `--purge` flag is used to indicate removing all Istio resources, including cluster scoped resources shared between different Istio control planes.

Reviewer 2: this should warn that the CRs that are  removed will remove all configuration data for the Isito control plane.

Author: Do you mean putting this in a warning block or the description is not clear enough?

Author: Added as a warning block

Reviewer 3: i think steve might have meant saying something like, "including cluster scoped resources that may still be used by another Istio version, and Istio user configuration. Use only if you are sure you want to completely remove every Istio related resource from the cluster."

## Uninstall Istio

> To uninstall Istio completely, run the following command:

Reviewer 4: ```suggestion
To completely uninstall Istio from a cluster, run the following command:
```

Author: done

{{< text bash >}}
$ istioctl x uninstall --purge
{{< /text >}}

{{< warning >}}

> The optional `--purge` flag indicates removal of all Istio resources, including cluster-scoped resources shared between different Istio control planes.

Reviewer 4: ```suggestion
The optional `--purge` flag will remove all Istio resources, including cluster-scoped resources that may be shared with other Istio control planes.
```

Author: done

> The optional `--purge` flag is used to indicate removing all Istio resources, including cluster scoped resources shared between different Istio control planes.

Author: But more precisely it is not the `--purge` flag does the operation, instead it is the uninstall command with `purge` flag

{{< /warning >}}

> Alternatively, you can run

Author: I changed from `specific installation manifest` to `specific Istio control plane`, which seems more accurate
